docs/manage/software/update-software.md

```diff
@@ -11,8 +11,8 @@ aliases:
 ---
 
 If you have already [deployed software](/manage/software/deploy-software/), you can inspect the fragment you have created.
-The JSON object for the deployed {{< glossary_tooltip term_id="module" text="module" >}} or {{< glossary_tooltip term_id="package" text="package" >}} has a `version` field.
-Unless the `version` field is set to a specific version, some or all updates for that module or package can happen automatically.
+Each deployed {{< glossary_tooltip term_id="module" text="module" >}} or {{< glossary_tooltip term_id="package" text="package" >}} has a `version` field.
+Unless the `version` field is set to a specific version, some or all updates for that module or package happen automatically.
```

Comment:
```suggestion
Unless the `version` field is set to a specific version, updates for that module or package happen automatically.
```
[q] 'some or all' is vague enough that I'm not sure it contributes anything to the sentence. not a huge deal, but the sentence is more readable without it.